### PR TITLE
[jsk_fetch_startup] Add yaml to set image compress png_level to 5

### PR DESCRIPTION
**Pull request by 708yamaguchi**
From https://github.com/knorth55/jsk_robot/pull/246

According to https://github.com/start-jsk/jsk_apc/pull/2706 and [gist.github.com/iory/12dda33908024223be74cb03f19997b5](https://gist.github.com/iory/12dda33908024223be74cb03f19997b5), set image compress `png_level` to 5.
This avoids low publish frequency of original image topics.

I checked the rosparams by the following command.
```
for f in $(rosparam list | grep png_level); do echo "$f: $(rosparam get $f)"; done
```

### CODE REVIEW

**Comment by knorth55** [comment]
Do we need to set `png_level` for `RGB` too?
In my understandings, depth takes long time because it's data type is `float16` or something.
`RGB` is `uint8` and has not that big effect, I thought.
What do you think?

**Comment by 708yamaguchi** [comment]
@knorth55 
Thank you very much for your advice.

> RGB is uint8 and has not that big effect, I thought.

I learned the effect of rgb compress.
I think rgb (uint8) compress has not big effect, so I remove it.
[`df7757f` (#1493)](https://github.com/jsk-ros-pkg/jsk_robot/pull/1493/commits/df7757f330dae27756e147b53ba7cb89f0d8f1db)

After that, I was advised by @iory and I added `disable_pub_plugins` for the following cases.
- compressedDepth for RGB images
- theora for depth images
- compressed for depth images

[`9cd9ee2` (#1493)](https://github.com/jsk-ros-pkg/jsk_robot/pull/1493/commits/9cd9ee2d489054269ff7218935f9c19feb20c664) and [`7ef9cc9` (#1493)](https://github.com/jsk-ros-pkg/jsk_robot/pull/1493/commits/7ef9cc9eb0f50d51d77c6325566dd0032cd9a887)

With this PR, the result of `rostopic list | grep head_camera` is the following.
```
fetch@fetch1075:~$ rostopic list | grep head_camera

/head_camera/crop_decimate/parameter_descriptions
/head_camera/crop_decimate/parameter_updates
/head_camera/depth/camera_info
/head_camera/depth/image
/head_camera/depth/image/compressedDepth
/head_camera/depth/image/compressedDepth/parameter_descriptions
/head_camera/depth/image/compressedDepth/parameter_updates
/head_camera/depth/image_raw
/head_camera/depth/image_raw/compressedDepth
/head_camera/depth/image_raw/compressedDepth/parameter_descriptions
/head_camera/depth/image_raw/compressedDepth/parameter_updates
/head_camera/depth/image_rect
/head_camera/depth/image_rect/compressedDepth
/head_camera/depth/image_rect/compressedDepth/parameter_descriptions
/head_camera/depth/image_rect/compressedDepth/parameter_updates
/head_camera/depth/image_rect_raw
/head_camera/depth/image_rect_raw/compressedDepth
/head_camera/depth/image_rect_raw/compressedDepth/parameter_descriptions
/head_camera/depth/image_rect_raw/compressedDepth/parameter_updates
/head_camera/depth/points
/head_camera/depth_downsample/camera_info
/head_camera/depth_downsample/image_raw
/head_camera/depth_downsample/image_raw/compressedDepth
/head_camera/depth_downsample/image_raw/compressedDepth/parameter_descriptions
/head_camera/depth_downsample/image_raw/compressedDepth/parameter_updates
/head_camera/depth_downsample/points
/head_camera/depth_rectify_depth/parameter_descriptions
/head_camera/depth_rectify_depth/parameter_updates
/head_camera/depth_registered/camera_info
/head_camera/depth_registered/hw_registered/image_rect
/head_camera/depth_registered/hw_registered/image_rect/compressedDepth
/head_camera/depth_registered/hw_registered/image_rect/compressedDepth/parameter_descriptions
/head_camera/depth_registered/hw_registered/image_rect/compressedDepth/parameter_updates
/head_camera/depth_registered/hw_registered/image_rect_raw
/head_camera/depth_registered/hw_registered/image_rect_raw/compressedDepth
/head_camera/depth_registered/hw_registered/image_rect_raw/compressedDepth/parameter_descriptions
/head_camera/depth_registered/hw_registered/image_rect_raw/compressedDepth/parameter_updates
/head_camera/depth_registered/image
/head_camera/depth_registered/image/compressedDepth
/head_camera/depth_registered/image/compressedDepth/parameter_descriptions
/head_camera/depth_registered/image/compressedDepth/parameter_updates
/head_camera/depth_registered/image_raw
/head_camera/depth_registered/image_raw/compressedDepth
/head_camera/depth_registered/image_raw/compressedDepth/parameter_descriptions
/head_camera/depth_registered/image_raw/compressedDepth/parameter_updates
/head_camera/depth_registered/image_rect
/head_camera/depth_registered/image_rect/compressedDepth
/head_camera/depth_registered/points
/head_camera/depth_registered/quater/points
/head_camera/depth_registered/quater/throttled/points
/head_camera/depth_registered/resize_cloud/input/mask
/head_camera/depth_registered/resize_cloud/parameter_descriptions
/head_camera/depth_registered/resize_cloud/parameter_updates
/head_camera/depth_registered/throttle_camera_info/parameter_descriptions
/head_camera/depth_registered/throttle_camera_info/parameter_updates
/head_camera/depth_registered/throttle_cloud/parameter_descriptions
/head_camera/depth_registered/throttle_cloud/parameter_updates
/head_camera/depth_registered/throttle_image/parameter_descriptions
/head_camera/depth_registered/throttle_image/parameter_updates
/head_camera/depth_registered/throttle_image_compressed/parameter_descriptions
/head_camera/depth_registered/throttle_image_compressed/parameter_updates
/head_camera/depth_registered/throttled/camera_info
/head_camera/depth_registered/throttled/image_rect
/head_camera/depth_registered/throttled/image_rect/compressedDepth
/head_camera/depth_registered_rectify_depth/parameter_descriptions
/head_camera/depth_registered_rectify_depth/parameter_updates
/head_camera/driver/parameter_descriptions
/head_camera/driver/parameter_updates
/head_camera/head_camera_nodelet_manager/bond
/head_camera/ir/camera_info
/head_camera/ir/image
/head_camera/projector/camera_info
/head_camera/rgb/camera_info
/head_camera/rgb/downsample_half/parameter_descriptions
/head_camera/rgb/downsample_half/parameter_updates
/head_camera/rgb/downsample_quater/parameter_descriptions
/head_camera/rgb/downsample_quater/parameter_updates
/head_camera/rgb/half/camera_info
/head_camera/rgb/half/image_rect_color
/head_camera/rgb/half/image_rect_color/compressed
/head_camera/rgb/half/image_rect_color/compressed/parameter_descriptions
/head_camera/rgb/half/image_rect_color/compressed/parameter_updates
/head_camera/rgb/half/image_rect_color/theora
/head_camera/rgb/half/image_rect_color/theora/parameter_descriptions
/head_camera/rgb/half/image_rect_color/theora/parameter_updates
/head_camera/rgb/image_raw
/head_camera/rgb/image_raw/compressed
/head_camera/rgb/image_raw/compressed/parameter_descriptions
/head_camera/rgb/image_raw/compressed/parameter_updates
/head_camera/rgb/image_raw/theora
/head_camera/rgb/image_raw/theora/parameter_descriptions
/head_camera/rgb/image_raw/theora/parameter_updates
/head_camera/rgb/image_rect_color
/head_camera/rgb/image_rect_color/compressed
/head_camera/rgb/image_rect_color/compressed/parameter_descriptions
/head_camera/rgb/image_rect_color/compressed/parameter_updates
/head_camera/rgb/image_rect_color/theora
/head_camera/rgb/image_rect_color/theora/parameter_descriptions
/head_camera/rgb/image_rect_color/theora/parameter_updates
/head_camera/rgb/quater/camera_info
/head_camera/rgb/quater/image_rect_color
/head_camera/rgb/quater/image_rect_color/compressed
/head_camera/rgb/quater/image_rect_color/compressed/parameter_descriptions
/head_camera/rgb/quater/image_rect_color/compressed/parameter_updates
/head_camera/rgb/quater/image_rect_color/theora
/head_camera/rgb/quater/image_rect_color/theora/parameter_descriptions
/head_camera/rgb/quater/image_rect_color/theora/parameter_updates
/head_camera/rgb/throttle_camera_info/parameter_descriptions
/head_camera/rgb/throttle_camera_info/parameter_updates
/head_camera/rgb/throttle_rgb/parameter_descriptions
/head_camera/rgb/throttle_rgb/parameter_updates
/head_camera/rgb/throttle_rgb_compressed/parameter_descriptions
/head_camera/rgb/throttle_rgb_compressed/parameter_updates
/head_camera/rgb/throttled/camera_info
/head_camera/rgb/throttled/image_rect_color
/head_camera/rgb/throttled/image_rect_color/compressed
/head_camera/rgb_rectify_color/parameter_descriptions
/head_camera/rgb_rectify_color/parameter_updates
/head_camera_fetch_recorder/rgb/image_rect_color
/head_camera_fetch_recorder/rgb_image_transport/compressed/parameter_descriptions
/head_camera_fetch_recorder/rgb_image_transport/compressed/parameter_updates
```

File: jsk_fetch_robot/jsk_fetch_startup/config/image_transport_plugins_params.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Check each node png_level by the following command
+# for f in $(rosparam list | grep png_level); do echo "$f: $(rosparam get $f)"; done
+
+# Set png_level to 5 according to the following information
+# https://github.com/start-jsk/jsk_apc/pull/2706
+# https://gist.github.com/iory/12dda33908024223be74cb03f19997b5
+
+burnable_trashbin_label_extractor:
```
[the remark this answers is not in the document]
Comment (708yamaguchi) [reply jsk_fetch_robot/jsk_fetch_startup/config/image_transport_plugins_params.yaml]: Thank you very much for your review.

I updated.
[`7ecbc40` (#1493)](https://github.com/jsk-ros-pkg/jsk_robot/pull/1493/commits/7ecbc406ead26f8f26ba052e2dfb8733d76d0a96)

**Review by knorth55 — APPROVED** [review]

**Comment by 708yamaguchi** [comment]
@knorth55 

Thank you very much for Merging branch 'master'.

@k-okada 

All tests have passed!

**Pull request merged**